tests/e2e/specs/wcpay/shopper/shopper-klarna-checkout.spec.js

```diff
@@ -25,11 +25,8 @@ describe( 'Klarna checkout', () => {
 		await shopperWCP.changeAccountCurrencyTo( 'USD' );
 	} );
 
-	afterEach( async () => {
-		await shopperWCP.emptyCart();
-	} );
-
 	afterAll( async () => {
+		await shopperWCP.emptyCart();
```

Comment:
QQ: why move the `emptyCart()` from `afterEach` to `afterAll`? I initially placed it there so that each test would be "atomic"/not dependent from the test runs on the same file.

Reply:
I did that because the first test, doesn't make use of cart. So before the first test, we don't necessarily need to empty the cart. For the second test, we add items to cart, hence I added empty cart to afterAll. And also inside the test. 

If you prefer, we can move it to beforeAll & afterAll, which will act the same.